### PR TITLE
New PR template

### PR DESCRIPTION
**Pull request by michellebieger**
A new PR template for this repository. Less onerous than the `twinLab` repository, but provides some more structure to encourage communication.

### CODE REVIEW

File: 00-quickstart.ipynb
Comment (alexander-mead) [inline 00-quickstart.ipynb:1]: This should probably be unchanged, but whatever!

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+_Who has been involved in completing the work?_
+
+## ✅ Checks
+
+_Mark the following items as complete when applicable:_
+
+- [ ] 🆕 Up to date with target branch
+- [ ] 🉑 Jira acceptance criteria met
+- [ ] 📚 Documentation updated
```
Comment (alexander-mead) [inline .github/pull_request_template.md:19]:
```suggestion
- [ ] 📚 Documentation updated
- [ ] 🧪 twinLab version requirement updated in `pyproject.toml`
```

**Review by alexander-mead — APPROVED** [review]
Great. One change and this is good to go!

File: 00-quickstart.ipynb
Comment (michellebieger) [reply 00-quickstart.ipynb:1]: Resolved, not sure how this slipped my notice. Apologies!

**Pull request merged**